src/python/pants/jvm/resolve/common.py

```diff
@@ -169,6 +173,17 @@ def from_jvm_artifact_target(cls, target: Target) -> ArtifactRequirement:
                 if target[JvmArtifactJarSourceField].value
                 else None
             ),
+            excludes=frozenset(target[JvmArtifactExcludeDependenciesField].value or []) or None,
+        )
+
+    def exclude(self, *excludes: str) -> ArtifactRequirement:
```

Comment:
Would `with_excludes` be a better name? I realize that it cannot be called `excludes` since that is a field, but using the singular for the method name has the potential to be very confusing vis-a-vis the field's name.

Reply:
Sure.